src/stp/new_plays/GenericPass.cpp

```diff
@@ -41,21 +41,21 @@ GenericPass::GenericPass() : Play() {
                                                                                  std::make_unique<role::Halt>(role::Halt("halt_7"))};
 }
 
-uint8_t GenericPass::score(world_new::World* world) noexcept { return 50; }
+uint8_t GenericPass::score(world_new::World* world) noexcept { return 100; }
 
 void GenericPass::calculateInfoForRoles() noexcept {
     // Keeper
     stpInfos["keeper"].setPositionToShootAt(Vector2{0.0, 0.0});
     stpInfos["keeper"].setEnemyRobot(world->getWorld()->getRobotClosestToBall(world_new::them));
 
-    const Vector2 passingPosition = calculatePassLocation();
+    const Vector2 passingPosition = Vector2(-field.getFieldLength()/2, -field.getFieldWidth()/2);//calculatePassLocation();
```

Comment:
No, this type of unclarity & code cloning does not happen under my watch. :disappointed: What you actually should use is: field.getBottomLeftCorner(). In general you should never encode position in this way, but create a separate definition in the field class for these special locations.